static/templates/subscription_settings.handlebars

```diff
@@ -21,10 +21,8 @@
                 {{#if is_admin}}
                 <button class="button small rounded btn-danger deactivate" type="button" name="delete_button" title="{{t 'Delete stream'}}">{{t 'Delete' }} <i class="icon-vector-trash" aria-hidden="true"></i></button>
                 {{/if}}
-                {{#if should_display_subscription_button}}
-                <button class="button small rounded subscribe-button sub_unsub_button {{#unless subscribed }}unsubscribed{{/unless}}" type="button" name="button" title="{{t 'Toggle subscription'}} (S)">
+                <button class="button small rounded subscribe-button sub_unsub_button {{#unless subscribed }}unsubscribed{{/unless}}" type="button" name="button" title="{{t 'Toggle subscription'}} (S)" {{#unless should_display_subscription_button}}style="display: none"{{/unless}}>
```

Comment:
I think we should clean all handlebars file which has conditional include of elements and has possibility of display changes in future. i.e. We don't need to hide stream display button, it can be done with conditionally(private stream icon or public stream icon), but `view stream`, `subscribe` buttons display can be changed in future.